codalab/lib/codalab_manager.py

```diff
@@ -127,8 +127,13 @@ def __init__(self, temporary=False, config={}, clients={}):
 
         if self.temporary:
             self.config = config
-            self.state = {'auth': {}, 'sessions': {}}
             self.clients = clients
+
+            # Initialize `state` with the content in `self.state.path`, if it exists.
+            if os.path.exists(self.state_path):
```

Comment:
Why is the codalab manager temporary when running the scenario in https://github.com/codalab/codalab-worksheets/issues/3888?

Reply:
Because of https://github.com/codalab/codalab-worksheets/pull/3151